### PR TITLE
Fix widget synchonization issue and CI build

### LINKED ISSUE #56: slider in cell not updating
Congrats on a release! I'm using the binder at https://mybinder.org/v2/gh/QuantStack/ipysheet/stable?filepath=examples.

I notice that when I run the first two cells of widget_cell notebook, a couple of things happen:

(a) the widget in the ipysheet cell resizes as I change it's value.
(b) When I move the slider in the first notebook cell (outside of ipysheet), the slider in ipysheet doesn't update, but the value in ipysheet does update.

screenshot

### PR DESCRIPTION
**Pull request by martinRenou**
This should fix #56

### CODE REVIEW

File: js/src/sheet.ts [hunk 1]
```diff
@@ -308,7 +308,7 @@ let SheetView = widgets.DOMWidgetView.extend({
         for (let key in this.widget_views) {
             if(this.widget_views.hasOwnProperty(key)) {
                 // Ugly, this should be properly done
-                let [row, col] = String(key).split(',').map(parseInt);
+                let [row, col] = String(key).split(',').map(x => parseInt(x));
```
Comment (jasongrout) [inline js/src/sheet.ts]: Is the problem that map calls parseInt with a lot of arguments, but really we just want parseInt to be called with the first argument, just the value?

Comment (jasongrout) [reply js/src/sheet.ts]: Oh wow, parseInt was interpreting the value as having radix whatever the index in the list was? Tricky!

Comment (martinRenou) [reply js/src/sheet.ts]: I don't understand what is going on on this. I only found the issue but don't understand it:
map_parseint

I don't think it has something to do with radix, does it? I don't really understand what you understood :P

Comment (martinRenou) [reply js/src/sheet.ts]: Ok, reading the documentation of `Array.map` I understand now :) That was tricky indeed :P

**Pull request merged**